### ISSUE TITLE
Fasta file not found

### ISSUE THREAD

**Issue by ggabernet** [nf-core/rnafusion#60]
I am getting the following error when running the pipeline:

```
N E X T F L O W  ~  version 19.01.0
Launching `nf-core/rnafusion` [evil_poisson] - revision: 12e5cb28cc [master]
ERROR ~ Fasta file not found: /sfs/4/sfs7/qbic/reference_genomes/Homo_sapiens/DNA/Ensembl_GRCh38_r90/sequence/GRCh38.r92.all.fasta`
```

The command I use to run it is the following:

```
nextflow run nf-core/rnafusion -r 1.0 --reads '*.{1,2}.fastq.gz' --star_index '/sfs/4/sfs7/qbic/reference_genomes/Homo_sapiens/DNA/Ensembl_GRCh38_r90/star' --fasta '/sfs/4/sfs7/qbic/reference_genomes/Homo_sapiens/DNA/Ensembl_GRCh38_r90/sequence/GRCh38.r92.all.fasta' --gtf '/sfs/4/sfs7/qbic/reference_genomes/Homo_sapiens/DNA/Ensembl_GRCh38_r90/annotation/Homo_sapiens.GRCh38.92.gtf' --star_fusion_ref '/sfs/7/workspace/ws/qeaga01-MTB_1143_fusion-0/star_fusion_ref' --star_fusion -profile cfc
```

The file exists at this path, I do have read rights and it is also in the singularity path. I'm not sure if this is a problem of our system or the pipeline but I thought I would ask if you have seen something like this before and how could I solve it.

Thanks for the great pipeline, I look forward to trying it :)

**Comment by matq007** [comment]
Hi @ggabernet I wasn't able to reproduce your error that you have. This error rises from this [line](https://github.com/nf-core/rnafusion/blob/12e5cb28cc05197e00cd900d4fc92e36a66be772/main.nf#L129). It is a standard check if the file exists. Hope you can solve the issue.

**Comment by matq007** [comment]
One more thing that I have realized, if you want to run `STAR-Fusion` you don't have to specify `fasta` or `gtf`. Those are only required if you want to build a STAR-Index from scratch. Since you have defined the path for the STAR-Index you just have to run:
`nextflow run nf-core/rnafusion -r 1.0 --reads '*.{1,2}.fastq.gz' -profile cfc --star_index '/sfs/4/sfs7/qbic/reference_genomes/Homo_sapiens/DNA/Ensembl_GRCh38_r90/star' --star_fusion_ref '/sfs/7/workspace/ws/qeaga01-MTB_1143_fusion-0/star_fusion_ref' --star_fusion`.

**Comment by ggabernet** [comment]
Hi Martin,
thanks a lot for checking this. In the meantime I found the "fasta file not found error" was my mistake, sorry for this!

However trying what you suggested in your last command without providing the fasta and gtf files raises an error:

'''
N E X T F L O W  ~  version 19.01.0
Launching `nf-core/rnafusion` [jovial_banach] - revision: 12e5cb28cc [1.0]
ERROR ~ You have to specify either fasta file or Genome version!
'''

**Comment by matq007** [comment]
Ah, you are right, this is my mistake. I haven't account for this situation when fasta and gtf is not required. I will update in the new upcoming version 1.0.1 that is almost ready. Thank you 👍

**Comment by ggabernet** [comment]
No worries, thank you!

**Comment by matq007** [comment]
Should be fixed in #61.
